server/app.py

```diff
@@ -868,7 +868,15 @@ def customer_subscription_created(event):
     # It starts by looking for a matching Contact (or creating one).
     subscription_id = event["data"]["object"]["id"]
     subscription = stripe.Subscription.retrieve(subscription_id, expand=["latest_invoice"])
-    donation_type = subscription["metadata"].get("donation_type", subscription["plan"]["metadata"].get("type", "membership"))
+    sub_meta = subscription["metadata"]
```

Comment:
**suggestion (non-blocking):** For readability/developer experience, it'd be more clear to me if this were `subscription_meta` to be consistent with the lines above. That could also be that I'm just jumping into this tho!

Reply:
yeah, I like that... fixed